ci/ansible/roles/pulp-crane/files/pulp_crane_el6.conf

```diff
@@ -0,0 +1,16 @@
+# source:
+# https://raw.githubusercontent.com/pulp/crane/3c657fe53020b804ce60d907a82b4fb19aebe532/deployment/apache22.conf
+# Use this config with Apache 2.2 or earlier
+Listen 5000
+<VirtualHost *:5000>
+    WSGIScriptAlias / /usr/share/crane/crane_el6.wsgi
+    <Location /crane>
+        Order deny,allow
+        Deny from all
+        Allow from localhost
```

Comment:
> Maybe the docker client itself only ends up trying to get data in `/usr/share/crane` ?

Aye. It's probably the case that Docker doesn't need to access the `/crane/repositories` path, so I over-stated the importance of this change in that regard.

The path is super-helpful in tests. The tests in `pulp_smash.tests.docker.cli.test_sync_publish` do use paths like `/crane/repositories`.